internal/operators/manager.go

```diff
@@ -120,10 +122,36 @@ func (mgr *Manager) GetRequirementsBreakdownForHostInCluster(ctx context.Context
 	return requirements, nil
 }
 
+func getStorageOperatorName(cluster *models.Cluster) string {
+	if common.HasOperator(cluster, odf.Operator.Name) {
+		return odf.Operator.Name
+	}
+	if common.HasOperator(cluster, lvm.Operator.Name) {
+		return lvm.Operator.Name
+	}
+	return ""
+}
+
+func getStorageClassName(cluster *models.Cluster) string {
+	if common.HasOperator(cluster, odf.Operator.Name) {
+		return odf.GetStorageClassName()
+	}
+	if common.HasOperator(cluster, lvm.Operator.Name) {
+		return lvm.GetStorageClassName()
+	}
+	return ""
+}
```

Comment:
I've tried a different approach now, but we still have a lengthy function to retrieve the storage operator.
The issue is that it could be the case that both are selected (unlikely but it's not forbidden), and we need the manager to pick one.
WDYT?